packages/sdk-react/src/core/hooks/useFormPersist.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { SetFieldValue } from 'react-hook-form';
+
+// Persists form state between page reloads
+
+export const useFormPersist = (
+  name: string,
+  getValues: () => { [key: string]: any },
+  setValue: SetFieldValue<any>
```

Comment:
SetFieldValue<any> is broad; consider narrowing this type to improve type safety